### PR TITLE
#81 timezone rectification

### PR DESCRIPTION
**Pull request by Skn0tt**
Closes: #81

## PR Checklist:

- [ ] PR has a descriptive title
- [ ] tests are passing
- [ ] branch has a descriptive name
- [ ] PR is structured into distinct & easy-to-review commits

## Pull Request Guidelines:

Make sure that:

a. You have done your changes in a separate branch. Branches MUST have descriptive names prefixed with according to the convention outlined [here](https://medium.com/tarkalabs/git-branch-naming-convention-60af30cd9a07). Good examples are: `bugfix/signin-issue`, `feature/google-calendar-integration` or `chore/issue-templates`.

b. Your commit messages are concise & descriptive.
Non-trivial changes can be explained in the commit body. (if not, use interactive rebase to rewrite history)

c. Your PR is thoughtfully structured into distinct commits, making it easy to review. (if not, use interactive rebase to rewrite history)

d. All tests are passing. If they don't, fix them first and amend to your commit (`git commit --amend`).

Please review the [CONTRIBUTING.md](../CONTRIBUTING.md) file for detailed contributing guidelines.

### CODE REVIEW

File: app/time-utils/format.ts [hunk 1]
```diff
@@ -1,3 +1,8 @@
 export function formatAs24HourClockString(date: Date, timeZone?: string) {
-  return new Intl.DateTimeFormat(undefined, { timeZone, timeStyle: "short" }).format(date)
+  return new Intl.DateTimeFormat(undefined, {
```
Comment (Skn0tt) [inline app/time-utils/format.ts:2]: This forces every user to see the same, European-style time format. I think we should adjust our tests, not the formatting.

File: app/meetings/components/schedules/tz.ts [hunk 2]
```diff
@@ -0,0 +1,399 @@
+export default `Africa/Abidjan
```
Comment (rohansaw) [inline app/meetings/components/schedules/tz.ts:1]: where and what is this file used for? Didn't see that it was imported anywhere?

Comment (Skn0tt) [reply app/meetings/components/schedules/tz.ts:1]: https://github.com/kalle-app/kalle/pull/153/files#diff-caf57dc7792a2843e4367de7aacea24dd440d2d7669bd2ab1b8a6719c3079bacR4

Comment (Skn0tt) [reply app/meetings/components/schedules/tz.ts:1]: https://github.com/kalle-app/kalle/pull/153/files#diff-4b50bb19d8547e3f10eccddb7cff19965bbfaa29c18642eef0846defdd612817R8

**Review by rohansaw — APPROVED** [review]

File: app/time-utils/format.ts (hunk 1 above)
Comment (phlprcks) [reply app/time-utils/format.ts]: yes, but if we want to do this, that needs to be changed in many places (e.g. for the schedule times as well). I suggest to work on this in a new isse (see #167)

**Comment by rohansaw** [comment]
@Skn0tt any idea why the test is not passing?

File: cypress/integration/meetings-flow.spec.ts [hunk 3]
```diff
@@ -43,19 +43,18 @@ it("Meetings Flow", () => {
   cy.get("#submit").click()
 
   cy.contains(url(`/schedule/${johnDoe.username}/${link}`))
-
   cy.visit(url(`/schedule/${johnDoe.username}/${link}`))
 
   cy.contains(format(dateToSelect, "d")).click()
-  cy.contains("10:30-11:00").click()
+  cy.contains("button", /10:30.*-11:00.*/)
```
Comment (Skn0tt) [inline cypress/integration/meetings-flow.spec.ts:49]:
```suggestion
  cy.contains("button", /10:30.*-11:00.*/).click()
```

**Pull request merged**